reqs/build.pip

```diff
@@ -1,4 +1,4 @@
-numpy<1.20
+numpy
```

Comment:
We want to build using an old version of numpy so we can support users with old versions of numpy.

Reply:
Sure there is few new cool stuff in numpy and a lot of ml space expect numpy >= 1.20 in thier repo, shouldn't maybe coreml legacy or LTS exist for stuff like that ❔